app/assess/templates/macros/justification.html

```diff
@@ -17,7 +17,7 @@ <h2 class="govuk-label-wrapper">
       </p>
     {% endif %}
     <textarea class="govuk-textarea" id="justification" name={{justification_form_name}} value=""
-        rows="10" aria-describedby="more-detail-hint"></textarea>
+        rows="10" aria-describedby="more-detail-hint" aria-label="Justification"></textarea>
```

Comment:
Can we use `label for` instead and link it to the label instead? e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Yes we can. Have pushed this change